asdi-resources/src/main/java/flyway/asdi2/V1_23_1__migrate_layer_list.java

```diff
@@ -0,0 +1,65 @@
+package flyway.asdi2;
+
+import fi.nls.oskari.util.PropertyUtil;
+import fi.nls.oskari.domain.map.view.Bundle;
+import fi.nls.oskari.log.LogFactory;
+import fi.nls.oskari.log.Logger;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.oskari.helpers.AppSetupHelper;
+import org.oskari.helpers.BundleHelper;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.List;
+
+public class V1_23_1__migrate_layer_list extends BaseJavaMigration {
+    private static final Logger LOG = LogFactory.getLogger(V1_23_1__migrate_layer_list.class);
+    private static final String LAYERSELECTOR_BUNDLE_NAME = "layerselector2";
+    private static final String LAYERSELECTION_BUNDLE_NAME = "layerselection2";
+    private static final String LAYERLIST_BUNDLE_NAME = "layerlist";
+    private static final String LAYER_EDITOR_BUNDLE_NAME = "admin-layereditor";
+
+    public void migrate(Context context) throws Exception {
+        if(PropertyUtil.getOptional("flyway.asdi.1_23_1.skip", true)) {
```

Comment:
Add a protected/public method that returns the boolean. That way this is easy to extend and override the toggle when we want this in an env where we skipped it the first time.